dune-project

```diff
@@ -1,2 +1,5 @@
-(lang dune 1.0)
+(lang dune 3.0)
 (name tls)
+(formatting disabled)
+(using mdx 0.2)
```

Comment:
could you tell what this does? `using mdx`? Does it introduce new dependencies?

Reply:
It uses MDX to run the tests (https://github.com/mirleft/ocaml-tls/blob/2a57f073373a8ae509a8dcd6b656e229c53fb516/eio/tests/tls_eio.md). It's only needed for testing tls-eio, which already lists it as a test dependency.

Reply:
ah cool, thanks!